### PR TITLE
Aligning top level modules readme titles to Start Case

### PR DESCRIPTION
**Pull request by eriqua**
# Change

Closes issue #651 
Including a couple of leftovers from previous workflow name alignment PR

## Type of Change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update (Wiki)

## Checklist

- [x] I'm sure there are no other open Pull Requests for the same update/change
- [x] My corresponding pipelines / checks run clean and green without any errors or warnings
- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (readme)
- [x] I did format my code

### CODE REVIEW

File: arm/Microsoft.ManagedIdentity/userAssignedIdentities/readme.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # User Assigned Identities `[Microsoft.ManagedIdentity/userAssignedIdentities]`
 
-This module deploys User Assigned Identities, with resource lock.
+This module deploys a user assigned identity, with resource lock.
```
Comment (MrMCake) [inline arm/Microsoft.ManagedIdentity/userAssignedIdentities/readme.md:3]: I'd propose to use the lock, or? Is it deployed by default?

File: arm/Microsoft.Network/expressRouteCircuits/readme.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
-# ExpressRoute Circuit `[Microsoft.Network/expressRouteCircuits]`
+# ExpressRoute Circuits `[Microsoft.Network/expressRouteCircuits]`
 
-This template deploys a ExrepressRoute Circuit.
+This template deploys an ExrepressRoute circuit.
```
Comment (MrMCake) [inline arm/Microsoft.Network/expressRouteCircuits/readme.md:3]: Typo & should be lower case

File: arm/Microsoft.Network/virtualWans/readme.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
-# Virtual Wan `[Microsoft.Network/virtualWans]`
+# Virtual WANs `[Microsoft.Network/virtualWans]`
 
-This template deploys Virtual Wan
+This template deploys a Virtual WAN.
```
Comment (MrMCake) [inline arm/Microsoft.Network/virtualWans/readme.md:3]: I guess we should again use lower case for `virtual`

File: arm/Microsoft.RecoveryServices/vaults/readme.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
-# RecoveryServicesVaults `[Microsoft.RecoveryServices/vaults]`
+# Recovery Services Vaults `[Microsoft.RecoveryServices/vaults]`
 
-This module deploys Recovery Service Vault, with resource lock.
+This module deploys a recovery service vault, with resource lock.
```
Comment (MrMCake) [inline arm/Microsoft.RecoveryServices/vaults/readme.md:3]: Same comment as before. I'd propose to remove the lock reference if it's not a default deployment

File: arm/Microsoft.Security/azureSecurityCenter/readme.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
-# AzureSecurityCenter `[Microsoft.Security/azureSecurityCenter]`
+# Azure Security Center `[Microsoft.Security/azureSecurityCenter]`
 
 This template enables Azure Security Center - Standard tier by default, could be overridden.
```
Comment (MrMCake) [inline arm/Microsoft.Security/azureSecurityCenter/readme.md:3]: Again we should probably use lower case

File: arm/Microsoft.Sql/managedInstances/readme.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 # SQL Managed Instances `[Microsoft.Sql/managedInstances]`
 
-This template deploys an SQL Managed Instance, with resource lock.
+This template deploys a SQL managed instance, with resource lock.
```
Comment (MrMCake) [inline arm/Microsoft.Sql/managedInstances/readme.md:3]: again a lock reference. See other comments

File: arm/Microsoft.Storage/storageAccounts/readme.md [hunk 7]
```diff
@@ -1,6 +1,6 @@
-# StorageAccounts `[Microsoft.Storage/storageAccounts]`
+# Storage Accounts `[Microsoft.Storage/storageAccounts]`
 
-This module is used to deploy an Azure Storage Account, with resource lock and the ability to deploy 1 or more Blob Containers and 1 or more File Shares. Optional ACLs can be configured on the Storage Account and optional RBAC can be assigned on the Storage Account and on each Blob Container and File Share.
+This module is used to deploy a storage account, with resource lock and the ability to deploy 1 or more blob containers, file shares, tables and queues. Optional ACLs can be configured on the storage account and optional RBAC can be assigned on the storage account and on each child resource.
 
 The default parameter values are based on the needs of deploying a diagnostic storage account.
```
Comment (MrMCake) [inline arm/Microsoft.Storage/storageAccounts/readme.md:5]: Is the sentence with the diagnostic storage account still true? Not sure what a storage account should be capable of to be called 'diagnostic storage account'

File: arm/Microsoft.Web/hostingEnvironments/readme.md [hunk 8]
```diff
@@ -1,6 +1,6 @@
 # App Service Environment `[Microsoft.Web/hostingEnvironments]`
 
-This module deploys App Service Environment, with resource lock.
+This module deploys an app service environment, with resource lock.
```
Comment (MrMCake) [inline arm/Microsoft.Web/hostingEnvironments/readme.md:3]: Again the lock reference. See other comments

Comment (eriqua) [reply arm/Microsoft.Web/hostingEnvironments/readme.md]: done

File: arm/Microsoft.Storage/storageAccounts/readme.md (hunk 7 above)
Comment (eriqua) [reply arm/Microsoft.Storage/storageAccounts/readme.md]: done

File: arm/Microsoft.RecoveryServices/vaults/readme.md (hunk 4 above)
Comment (eriqua) [reply arm/Microsoft.RecoveryServices/vaults/readme.md]: done

File: arm/Microsoft.ManagedIdentity/userAssignedIdentities/readme.md (hunk 1 above)
Comment (eriqua) [reply arm/Microsoft.ManagedIdentity/userAssignedIdentities/readme.md]: done

File: arm/Microsoft.Network/expressRouteCircuits/readme.md (hunk 2 above)
Comment (eriqua) [reply arm/Microsoft.Network/expressRouteCircuits/readme.md]: done

File: arm/Microsoft.Network/virtualWans/readme.md (hunk 3 above)
Comment (eriqua) [reply arm/Microsoft.Network/virtualWans/readme.md]: done

File: arm/Microsoft.Security/azureSecurityCenter/readme.md (hunk 5 above)
Comment (eriqua) [reply arm/Microsoft.Security/azureSecurityCenter/readme.md]: done

File: arm/Microsoft.Compute/galleries/readme.md [hunk 9]
```diff
@@ -1,6 +1,6 @@
-# Shared Image Gallery `[Microsoft.Compute/galleries]`
+# Azure Compute Galleries `[Microsoft.Compute/galleries]`
 
-This module deploys Share Image Gallery, with resource lock.
+This module deploys an azure compute gallery (formerly known as shared image gallery).
```
Comment (MrMCake) [inline arm/Microsoft.Compute/galleries/readme.md:3]: `Azure` must be upper case

File: arm/Microsoft.DesktopVirtualization/applicationgroups/readme.md [hunk 10]
```diff
@@ -1,6 +1,6 @@
 # AVD Application Groups `[Microsoft.DesktopVirtualization/applicationgroups]`
 
-This module deploys AVD Application Groups, with resource lock and diagnostics configuration.
+This module deploys AVD Application Groups.
```
Comment (MrMCake) [inline arm/Microsoft.DesktopVirtualization/applicationgroups/readme.md:3]:
```suggestion
This module deploys an AVD application groups.
```

File: arm/Microsoft.DesktopVirtualization/hostpools/readme.md [hunk 11]
```diff
@@ -1,6 +1,6 @@
-# AVD HostPools `[Microsoft.DesktopVirtualization/hostpools]`
+# AVD Host Pools `[Microsoft.DesktopVirtualization/hostpools]`
 
-This module deploys AVD Host Pools, with resource lock and diagnostics configuration.
+This module deploys a AVD Host Pool.
```
Comment (MrMCake) [inline arm/Microsoft.DesktopVirtualization/hostpools/readme.md:3]:
```suggestion
This module deploys an Azure virtual desktop host pool.
```

File: arm/Microsoft.DesktopVirtualization/workspaces/readme.md [hunk 12]
```diff
@@ -1,6 +1,6 @@
 # AVD Workspaces `[Microsoft.DesktopVirtualization/workspaces]`
 
-This module deploys AVD Workspaces, with resource lock and diagnostic configuration.
+This module deploys an AVD Workspace.
```
Comment (MrMCake) [inline arm/Microsoft.DesktopVirtualization/workspaces/readme.md:3]:
```suggestion
This module deploys an Azure virtual desktop workspace.
```

File: arm/Microsoft.Security/azureSecurityCenter/readme.md [hunk 13]
```diff
@@ -1,6 +1,6 @@
-# AzureSecurityCenter `[Microsoft.Security/azureSecurityCenter]`
+# Azure Security Center `[Microsoft.Security/azureSecurityCenter]`
 
-This template enables Azure Security Center - Standard tier by default, could be overridden.
+This template enables azure security center - Standard tier by default, could be overridden.
```
Comment (MrMCake) [inline arm/Microsoft.Security/azureSecurityCenter/readme.md:3]:
```suggestion
This template enables Azure security center - Standard tier by default, could be overridden.
```

**Review by MrMCake — APPROVED** [review]

**Pull request merged**